e2e/dpos-2-validators.toml

```diff
@@ -4,6 +4,13 @@
   All = true
   Condition = "contains"
   Expected = ["{{index $.NodeBase64AddressList 0}}", "{{index $.NodeBase64AddressList 1}}"]
+  Json = true
+  ExpectedJson = [
+  "statistics.#.address.chainId", "default",
+  "statistics.#.address.local", "{{index $.NodeBase64AddressList 0}}",
+  "statistics.#.address.chainId", "default",
+  "statistics.#.address.local", "{{index $.NodeBase64AddressList 1}}"
```

Comment:
Is there a way to match the first element & second element specifically? So we can make sure the first statistic matches node 0, and the second statistic matches node 1?

Reply:
Yes, to do that we need to add some query to the key. 
I already add some example of usage.